ouroboros-consensus-diffusion/src/Ouroboros/Consensus/Node.hs

```diff
@@ -178,13 +178,11 @@ data RunNodeArgs m addrNTN addrNTC blk (p2p :: Diffusion.P2P) = RunNodeArgs {
       -- | Network P2P Mode switch
     , rnEnableP2P :: NetworkP2PMode p2p
 
-<<<<<<< HEAD
       -- | Network PeerSharing miniprotocol willingness flag
     , rnPeerSharing :: PeerSharing
-=======
+
       -- | Whether to use the LMDB or the in-memory backend for UTxO-HD.
     , rnBackingStoreSelector :: !(BackingStoreSelector m)
->>>>>>> 2e9b596342 (Introduce LMDB backing store implementation (#4366))
```

Comment:
Maybe we should try to remove these merge conflicts in the commit where they orginate from

Reply:
I thought I had done that already :thinking:

Reply:
Ah right. I'm removing them here instead of removing them from where they come!

Reply:
Done